config/resources/domain.json

```diff
@@ -206,7 +206,7 @@
     },
     "snippet-lists": {
       "name": "snippet-list",
-      "class": "ext:SnippetList",
+      "class": "gn:SnippetList",
```

Comment:
For me the concept of snippets is one from the editor and how it functions, rather than being related to gelinkt notuleren. It would make sense to me to use the https://say.data.gift/ns/ namespace for these, unless we're trying to move away from that?

Reply:
Yes I guess using the say namespace is better here, what do you think @abeforgit ?

Reply:
yeah sounds good